zebrad/src/components/sync.rs

```diff
@@ -130,6 +130,17 @@ pub(super) const BLOCK_VERIFY_TIMEOUT: Duration = Duration::from_secs(180);
 /// previous sync runs.
 const SYNC_RESTART_DELAY: Duration = Duration::from_secs(61);
 
+/// Controls how long we wait to retry in a failed attempt to download
+/// and verify the genesis block.
+///
+/// This timeout gives the crawler time to find better peers.
+///
+/// ## Correctness
+///
+/// If this timeout is removed (or set too low), Zebra will retry
+/// inmediatly to download and verify the genesis block from its peers.
```

Comment:
This is a security issue, not a correctness issue, because Zebra can take down remote nodes.
```suggestion
/// Controls how long we wait to retry a failed attempt to download
/// and verify the genesis block.
///
/// This timeout gives the crawler time to find better peers.
///
/// ## Security
///
/// If this timeout is removed (or set too low), Zebra will immediately retry
/// to download and verify the genesis block from its peers. This can cause
/// a denial of service on those peers.
```